include/wally_psbt.h

```diff
@@ -0,0 +1,429 @@
+#ifndef LIBWALLY_CORE_PSBT_H
+#define LIBWALLY_CORE_PSBT_H
+
+#include "wally_transaction.h"
+#include "wally_core.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+const uint8_t WALLY_PSBT_MAGIC[5] = {'p', 's', 'b', 't', 0xff};
```

Comment:
shouldn't it be `extern const` with value itself defined in `psbt.c`?
otherwise if I include `wally_psbt.h` in two different files in the project I got linking problems with `duplicate symbol`...

Reply:
your analysis is correct. I suspect we may not want it at all in the header file at this stage either. regardless I'd suggest a function would be preferred as to simplify the bindings for the other languages.

Reply:
I've made it a `extern const`